db/bigtable_eth1.go

```diff
@@ -2102,8 +2102,10 @@ func (bigtable *Bigtable) GetAddressInternalTableData(address []byte, search str
 
 	names := make(map[string]string)
 	for _, t := range transactions {
-		names[string(t.From)] = ""
-		names[string(t.To)] = ""
+		if t != nil {
+			names[string(t.From)] = ""
+			names[string(t.To)] = ""
+		}
```

Comment:
This check does not suffice. A few lines below, `t.From` and `t.To` are accessed again.

By the way, this also applies to the check here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2f7566c921508e73a435184ee5d86c78258e95f3/db/bigtable_eth1.go#L1828
This wasn't added by you but has the same problem.